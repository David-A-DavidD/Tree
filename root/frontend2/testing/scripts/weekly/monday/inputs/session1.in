login
sellUser
sell
gameForSale
100
logout
exit