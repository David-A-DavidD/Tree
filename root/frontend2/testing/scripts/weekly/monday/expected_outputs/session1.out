Enter transaction code (or 'exit' to quit): Enter username: Login successful.
Enter the game name (up to 25 characters): Enter the price for the game (up to $999.99): Game listed for sale.
Logout successful.